.gitmodules

```diff
@@ -58,3 +58,7 @@
 	path = cmake/configs/uavcan_board_ident
 	url = https://github.com/PX4/uavcan_board_ident.git
 	branch = master
+[submodule "mavlink/include/mavlink/v2.0"]
```

Comment:
Change this the old one and create a PR there

Reply:
Done